src/main/java/jtdiff/antlr/YamlSerializer.java

```diff
@@ -82,11 +82,21 @@ public void visitTerminal(TerminalNode node) {
     }
 
     indent(mIndentCount, mStringBuilder);
-    mStringBuilder.append("- ")
-                  .append("'")  // Escape for Yaml
-                  .append(name)
-                  .append("'")  // Escape for Yaml
-                  .append(":\n");
+    
+    // To avoid exception caused by char values
+    if(name.contains("'")) {
+       mStringBuilder.append("- ")
+		     .append(name.contains(":") ? " ": name)
```

Comment:
Can we define name before the if block (It would also be better to be more specific - contains vs. equals) and add a TODO to remind us to remove this later?:

```
// TODO: remove this when we use newer version of apted library.
if (name.equals(":")) {
   name = "\semicol";
}
```

Can we define a boolean isChar to make what the decision is more explicit:

```
boolean notChar = !name.startsWith("'");
```

Then use that for comparison:

```
mStringBuilder.append("- ");
if (notChar) {
  mStringBuilder.append("'")  // Escape for Yaml
  mStringBuilder.append(name);
  mStringBuilder.append("'")  // Escape for Yaml
} else {  // Already "escaped"
  mStringBuilder.append(name);
}
mStringBuilder.append(":\n");
```